users/config.rst

```diff
@@ -305,6 +309,8 @@ Folder Element
         <junctionsAsDirs>false</junctionsAsDirs>
         <syncOwnership>false</syncOwnership>
         <scanOwnership>false</scanOwnership>
+        <syncXattrs>false</syncXattrs>
+        <scanXattrs>false</scanXattrs>
```

Comment:
```suggestion
        <sendXattrs>false</sendXattrs>
```

Reply:
By the way, there's still `scanOwnership` right above this 😉.

Reply:
That is already fixed by #766, which also causes the conflicts right now.